massa-bootstrap/src/lib.rs

```diff
@@ -46,13 +46,25 @@ pub struct GlobalBootstrapState {
     /// timestamp correction in milliseconds
     pub compensation_millis: i64,
 
-    /// list of network peers
-    pub peers: Option<BootstrapPeers>,
+    /// list of network and bootstrap peers, true if bootstrap
```

Comment:
You mean that the Boolean of the pair is true if the current node do have bootstrapped from this peer?

Reply:
no it means if this ip appeared in our bootstrap list